packages/document/main-doc/docs/en/guides/basic-features/alias.mdx

```diff
@@ -50,7 +52,7 @@ Modern.js also provides a way to config aliases. Adding the `@common` alias as a
 }
 ```
 
-JavaScript project can config by [`source.alias`](/configure/app/source/alias) in `modern.config.js`:
+JavaScript projects can configure [`source.alias`](/configure/app/source/alias) in `modern.config.js`:
```

Comment:
JavaScript projects can configure [`source.alias`](/configure/app/source/alias) in `modern.config.js`:
->

For JavaScript projects,  set [`source.alias`](/configure/app/source/alias) in `modern.config.js`: